MediaManager.Abstractions/IMediaTrackInfo.cs

```diff
@@ -0,0 +1,20 @@
+
+using System.ComponentModel;
+using Plugin.MediaManager.Abstractions.Enums;
+
+namespace Plugin.MediaManager.Abstractions
+{
+    public interface IMediaTrackInfo : INotifyPropertyChanged
```

Comment:
What are the extra properties for? There already is a TrackInfo class.

Reply:
Are you referring to the Android TrackInfo? https://developer.xamarin.com/api/type/Android.Media.MediaPlayer+TrackInfo/
This new class is to abstract TrackInfo at PCL level. Need this to implement the selection of audio channel.